### DISCUSSION TITLE
Scythe Anticheat 2.17.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#116] [Announcements]
Scythe is now at version v2.17.0!

Supported Minecraft versions: 1.20.40, 1.20.41

# Changelog
- Added support for MC version 1.20.40

By downloading this addon, you agree to our [license](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/blob/main/LICENSE)
Questions? Join our [discord](https://discord.gg/pKyu95mYCT)!

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/v2.17.0'>Scythe Anticheat 2.17.0</a>.</em>

**Comment by ExtraBytez** [comment]
You have to add support for new versions of Mc now?! For a mod that sure sucks is it a scripting update or smth

**Reply by MrDiamond64** [reply]
It's because of how Minecraft Scripting API works. Scythe uses the beta version of the Scripting API.

When Mojang changes the Scripting API, it requires the version of the beta module to be updated. As you can see in [this commit](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/commit/78e5e5c2964f4640fac3cf9d553b75c8aa4b17a3), the only change needed was to update the version of the Scripting API used.
